source/_UIAHandler.py

```diff
@@ -262,6 +262,11 @@ def IUIAutomationPropertyChangedEventHandler_HandlePropertyChangedEvent(self,sen
 		eventHandler.queueEvent(NVDAEventName,obj)
 
 	def _isUIAWindowHelper(self,hwnd):
+		if hwnd == winUser.getDesktopWindow():
+			# #6301: In Windows 10 build >= 14901,
+			# calling UiaHasServerSideProvider with the desktop window sometimes freezes when bringing up NVDA windows.
+			# It will always return False anyway.

```

Comment:
Can this be clarified to "Exit early when hwnd is the windows desktop handle, UiaHasServerSideProvider would return false anyway." ?